.github/pull_request_template.md

```diff
@@ -23,8 +23,8 @@ Fixes # (issue)
 -->
 
 - [ ] Bug fix
-- [ ] Test case(new/improvement)
```

Comment:
Can we keep this option as "Unit tests"?

Reply:
Added, thanks!